artiq/frontend/artiq_ctlmgr.py

```diff
@@ -60,25 +80,27 @@ def main():
         loop = asyncio.get_event_loop()
     atexit.register(loop.close)
 
-    logfwd = LogForwarder(args.server, args.port_logging,
-                          args.retry_master)
+    # Set up log forwarding to remote PC/master
+    logfwd = LogForwarder(args.server, args.port_logging, args.retry_master)
     logfwd.addFilter(source_adder)
     root_logger.addHandler(logfwd)
     logfwd.start()
     atexit_register_coroutine(logfwd.stop)
 
-    ctlmgr = ControllerManager(args.server, args.port_notify,
-                               args.retry_master)
+    # Start a Controller Manager. Waits to start devices until requested by Master.
```

Comment:
It's not really "requested" by the master.

Reply:
So how would you phrase it? This is not a helpful comment, and the code has no documentation or docstrings or comments to convey what is actually supposed to be happening.

Reply:
It is a sync_struct ``Subscriber``, so the master doesn't really request devices to be started, but initializes the subscriber's copy of the device database (and then sends change notifications).

Reply:
Removed comment.